### DISCUSSION TITLE
Unable to install packages from private npm registry.

### DISCUSSION THREAD

**Discussion by hiaw** [teambit/bit#8192] [Help]
I'm having a lot of trouble trying to do `bit install`. So my company has private npm packages that I'm dependent on. `pnpm install`, `yarn install` all work but `bit install` doesn't work. The error is show below. As you can see in the `npm config list`, the token for both bit and npm has been set correctly. 

![Screenshot 2023-11-27 at 5 42 57 PM](https://github.com/teambit/bit/assets/527296/dff5261c-136d-47c2-98f6-93f6004b0032)

```
~/D/D/V/_/frontend-northstar ❯❯❯ npm config list                                                main ✚ ◼
; "user" config from /Users/d/.npmrc

//registry.npmjs.org/:_authToken = (protected) 

; "project" config from /Users/d/Documents/Development/Veho/_test/frontend-northstar/.npmrc

@teambit:registry = "https://node-registry.bit.cloud" 
@vehodanielchong:registry = "https://node-registry.bit.cloud" 
//node-registry.bit.cloud/:_authToken = (protected) 
always-auth = true 

; node bin location = /Users/d/.asdf/installs/nodejs/18.17.1/bin/node
; node version = v18.17.1
; npm local prefix = /Users/d/Documents/Development/Veho/_test/frontend-northstar
; npm version = 9.6.7
; cwd = /Users/d/Documents/Development/Veho/_test/frontend-northstar
; HOME = /Users/danielchong
; Run `npm config ls -l` to show all defaults.
```

**Comment by ranm8** [comment]
Bit uses pnpm and your npmrc by default. I am not sure what went wrong here. @GiladShoham can you help?

**Comment by GiladShoham** [comment]
I don't see a custom registery configured for @veho which means it go to the default one that according to the output is bit's registry.
So either add a config for it or change the default one (then you might need to make it proxy to our registry)
